plugins/jenkins/api-report.md

```diff
@@ -72,7 +72,7 @@ export interface JenkinsApi {
     entity: EntityName;
     jobFullName: string;
     buildNumber: string;
-  }): Promise<void>;
+  }): Promise<Response>;
```

Comment:
Did you really mean to make these return a bare Response? Would it be clearer perhaps to do [something like this](https://github.com/backstage/backstage/blob/393f107893ec4ab4903d9022b8c3d0b6d0c4970d/packages/catalog-client/src/CatalogClient.ts#L394)? The next step up after that would be to throw a domain specific error type instead, but that's more work of course.

Reply:
You're right, I'm still learning the backstage ecosystem thanks for tipp :) I've updated the implementation to throw an error. I think it would be good to return response body to show some additional information but in this case I need to check the API about the body content. But for such improvements I can create another PR.